src/jvmTest/kotlin/mu/KotlinLoggingTest.kt

```diff
@@ -1,18 +1,20 @@
 package mu
 
+import mu.KotlinLogging.logger
```

Comment:
I think it makes things more confusing

Reply:
The function has to be imported directly in order for the reified inline function to be executed properly. If it were called using the object name (i.e. `val inlineLoggerInClass = KotlinLogging.logger()`) then the logger will be instantiated for `mu.KotlinLogging`, not for `mu.ForKotlinLoggingTest`.